src/main/java/com/spartronics4915/frc2020/RobotContainer.java

```diff
@@ -15,6 +17,8 @@
 
 public class RobotContainer
 {
+    // TODO: verify subsystem creation placement
```

Comment:
This TODO is confusing. 'placement'?

Reply:
Leaving it -- it is meant for @j-james as I didn't see where the subsystems are instantiated